diff --git a/meson.build b/meson.build
index 4766ce838ba..eb788fadd31 100644
--- a/meson.build
+++ b/meson.build
@@ -680,7 +680,7 @@ _va = get_option('gallium-va') \
 _dep_va_name = host_machine.system() == 'windows' ? 'libva-win32' : 'libva'
 dep_va = dependency(_dep_va_name, version : '>= 1.8.0', required : _va)
 if dep_va.found()
-  dep_va_headers = dep_va.partial_dependency(compile_args : true)
+  dep_va_headers = dep_va.partial_dependency(compile_args : true, includes: true)
   if cc.has_header_symbol('va/va.h', 'VASurfaceAttribDRMFormatModifiers',
                           dependencies: dep_va_headers)
     pre_args += '-DHAVE_VA_SURFACE_ATTRIB_DRM_FORMAT_MODIFIERS'
